gpt4all-chat/qml/Theme.qml

```diff
@@ -1077,6 +1108,7 @@ QtObject {
     property color codeKeywordColor: {
         switch (MySettings.chatTheme) {
             case "LegacyDark":
+                return "#195273"; // dark blue
```

Comment:
This changes the actual theme. Please make this a standalone PR and describe why you like these colors more.